internal/db/group.go

```diff
@@ -82,41 +82,30 @@ func (d *Database) GetGroup(ctx context.Context, group *dbmodel.GroupEntry) (err
 	return nil
 }
 
-// ForEachGroup iterates through every group calling the given function
-// for each one. If the given function returns an error the iteration
-// will stop immediately and the error will be returned unmodified.
-func (d *Database) ForEachGroup(ctx context.Context, limit, offset int, f func(*dbmodel.GroupEntry) error) (err error) {
+// ListGroups returns a paginated list of groups defined by limit and offset.
+// match is used to filter entries based on name.
```

Comment:
could we please add the expected format of match (is it a regexp?), or perhaps just provide examples

Reply:
updated godoc. I think it's clear now with the LIKE operator, if you'd like examples as well I can add them